fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/scalar/CallOperator.java

```diff
@@ -109,7 +120,18 @@ public void setChild(int index, ScalarOperator child) {
 
     @Override
     public boolean isNullable() {
-        return fn == null || fn.isNullable();
+        if (fn == null) {
+            return true;
+        }
+        // check if fn always return non null
+        if (!fn.isNullable()) {
+            return false;
+        }
+        // check children nullable
+        if (nullableSameWithChildrenFunctions.contains(fnName)) {
+            return arguments.stream().anyMatch(ScalarOperator::isNullable);
```

Comment:
I think you can't use child‘s nullable info, our nullable info is not accurate now

Reply:
done